memory/reflection.py

```
@@ -101,9 +110,43 @@ def save_reflections(self, name: str, reflections: list[dict]) -> None:
+        # 归档到独立文件
+        if to_archive:
+            archive_path = self._reflections_archive_path(name)
+            existing: list[dict] = []
+            if os.path.exists(archive_path):
+                try:
+                    with open(archive_path, encoding='utf-8') as f:
+                        data = json.load(f)
+                    if isinstance(data, list):
+                        existing = data
+                except (json.JSONDecodeError, OSError) as e:
+                    # 归档文件损坏 → 放弃本次归档，保留在主文件中，避免覆盖丢数据
```

Ruff (0.15.7)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)